WMass/python/plotter/w-helicity-13TeV/functionsWMass.cc

```diff
@@ -333,6 +333,9 @@ float residualScale(float pt, float eta, int isData) {
 }
 
 float ptElFull(float pt, float eta, int nSigma=0) {
+
+  if (nSigma == 0) return pt;
```

Comment:
is this needed? if nSigma=0 the function is already returning pt.
@cippy says that it optimizes 👍